src/utilities/formatters.js

```diff
@@ -17,3 +17,22 @@ export const formatExposureRange = ({ minTemperature, maxTemperature } = {}) =>
   const degree = String.fromCharCode(176);
   return `${minTemperature}${degree}C to ${maxTemperature}${degree}C`;
 };
+
+export const formattedDifferenceBetweenDates = (dateA, dateB) => {
+  const differenceInMs = new Date(dateB.getTime() - dateA.getTime()).getTime();
+
+  const msPerMinute = 1000 * 60;
+  const msPerHour = msPerMinute * 60;
+  const msPerDay = msPerHour * 24;
+
+  const numberOfDays = Math.floor(differenceInMs / msPerDay);
+  const numberOfHours = Math.floor((differenceInMs - numberOfDays * msPerDay) / msPerHour);
+  const numberOfMinutes = Math.floor((differenceInMs - numberOfHours * msPerHour) / msPerMinute);
+
+  let formattedDifference = '';
+  if (numberOfDays) formattedDifference += `${numberOfDays} Days `;
+  if (numberOfHours) formattedDifference += `${numberOfHours} Hours `;
+  if (numberOfMinutes) formattedDifference += `${numberOfMinutes} Minutes `;
```

Comment:
this method is fine for now, but i would envision showing something like this:
1 week/s, 2 day/s, 2 day/s 1 hour/s, 6 hour/s 12 minute/s, 200 minutes
we would need to set a constants (number of milliseconds, to switch between day hour or minutes)